libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2354,3 +2354,49 @@ std::string YulUtilFunctions::tryDecodeErrorMessageFunction()
 		).render();
 	});
 }
+
+string YulUtilFunctions::returndatacopyFunction()
+{
+	string const functionName = "returndata_copy";
+
+	if (!m_evmVersion.supportsReturndata())
```

Comment:
Has to go inside the `createFunction` call.

Did this not fail the tests?

Reply:
I think this does not fail because when you compile you compile for only *one* EVM version, you don't change unless you reuse your objects but change the EVM version without resetting YulUtilFunctions. We could verify this, hmm... or I just move it inside right away. :)

Reply:
The problem is that if you call this function multiple times, it will be defined muliple times.